05_node_operations.asciidoc

```diff
@@ -72,134 +72,134 @@ in the cloud. Nonetheless, if running your own server is not an option, by all m
+_MyNode_ is another popular open source "helper" project including a lot of Bitcoin related software. It is easy to install: you "flash" the installer onto an SD card and boot your mini-PC from the SD card. You do not need any monitor to use myNode as the administrative tools are accessible remotely from a browser. If your mini-PC has no monitor, mouse, or keyboard; you can manage it from another computer or even from your smartphone. Once installed, go to http://mynode.local/ and create a Lightning wallet and node in two clicks.
 
 You can find the MyNode project here:
 
 https://mynodebtc.com/
 
 In addition to a Bitcoin and Lightning node, MyNode can optionally install a variety of additional services, such as:
 
-- Ride The Lightning (Lightning Node Management GUI)
-- VPN Support for remote management or wallet (OpenVPN)
-- lndmanage (CLI management tool)
-- btc-rpc-explorer (A Bitcoin blockchain explorer)
+* Ride The Lightning (Lightning node management GUI)
+* OpenVPN (VPN Support for remote management or wallet)
+* LNDmanage (command-line interface for advanced channel management)
+* btc-rpc-explorer (A Bitcoin blockchain explorer)
 
 ==== Umbrel
 
-Famous for their UX/UI, _Umbrel_ provides a very easy and accessible way to get your Bitcoin and Lightning node up and running in no time, especially for beginners. A very distinctive feature is that _Umbrel_ utilizes Neutrino/SPV during the IBD (Inital Blockchain Download) so you can instantly start using your node. Once Bitcoin Core is fully synced in the background it automatically switches over and disables SPV mode. Umbrel OS supports the Raspberry Pi 4 and can also be installed on any Linux-based OS or on a virtual machine on macOS or Windows. You can also connect any wallet that supports Bitcoin Core P2P, Bitcoin Core RPC, the Electrum protocol or lndconnect.
+Famous for their UX/UI, _Umbrel_ provides a very easy and accessible way to get your Bitcoin and Lightning node up and running in no time, especially for beginners. A very distinctive feature is that _Umbrel_ utilizes Neutrino/SPV during the Inital Blockchain Download so you can instantly start using your node. Once Bitcoin Core is fully synced in the background it automatically switches over and disables SPV mode. Umbrel OS supports the Raspberry Pi 4 and can also be installed on any Linux-based OS or on a virtual machine on macOS or Windows. You can also connect any wallet that supports Bitcoin Core P2P, Bitcoin Core RPC, the Electrum protocol or lndconnect.
 
-No need to wait for a rainy day, you can find the project here: https://getumbrel.com
+There's no need to wait for a rainy day - you can find the project here: https://getumbrel.com
 
 image::images/umbrel.png[]
 
 In addition to a Bitcoin and Lightning node, Umbrel introduced the Umbrel App Store, where you can easily install additional services, such as:
 
-* Lightning Terminal (Interface for managing channel liquidity, loop-in & loop-out)
```

Comment:
As above capitalization here is taken from another source.  My suggested changes are more accurate.